pkgs/tools/misc/hostmux/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, installShellFiles, openssh, tmux, ... }:
+
+stdenv.mkDerivation rec {
+  version = "1.4.0";
+  pname = "hostmux";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  buildInputs = [ openssh tmux ];
+
+  src = fetchFromGitHub {
+    owner = "hukl";
+    repo = pname;
+    rev = version;
+    sha256 = "0h1wcbzlckjzg0y2lljf840kxvqzv5c6snn5bcbcpp0lbd07plx1";
```

Please use SRI hash.

```suggestion
    hash = "sha256-odN7QFsU3MsWW8VabVjZH+8+AUFOUio8eF9ORv9iPEA=";
```